simple/sample/output/triples.csv

```diff
@@ -0,0 +1,9 @@
+subject_id,predicate,object_id,object_value
```

Comment:
Mixer now can only read from one "raw data" folder, so the triples csv would ideally be in the same folder as the observations.

Reply:
Triples csv is in the same folder as observations. Basically, the same folder where you read the observations to load the sqlite db.

Reply:
Wonder why the triples.csv here is not in the same folder of observations?

Reply:
It is in the same folder: https://screenshot.googleplex.com/8G86oAmfkmgU4ht

If you're implying a different folder, let me know.